### ISSUE TITLE
Toon Cup 2013: Cannot start game

### ISSUE THREAD

**Issue by n0samu** [ruffle-rs/ruffle#11887]
### Describe the bug

Link to SWF file: https://flashga.me/public/storage/games/toon-cup-2013-files/launcher-fixed.swf
(Note: This is a multiasset game so it needs to be loaded via its URL)

In Ruffle, [Toon Cup 2013](https://flashga.me/toon-cup-2013.html) gets stuck on the loading screen. Here are some of the log messages:
```
WARN ruffle_core::stub: Encountered stub: AVM2 XML.name() namespaces
ERROR::LoaderManager>> Type no found '' url : demo/
... many more instances of the above error ...
INFO avm_trace: [object Application] >> Application is loaded :: 0 bytesLoaded
ERROR ruffle_core::avm2::globals::flash::net::shared_object: SharedObject::get_local: Movie was None
INFO avm_trace: ERROR::[object Application]>> getXMLData [strings] no found
ERROR ruffle_core::avm2::events: Error dispatching event EventObject(EventObject { type: "complete", class: flash.events::Event, ptr: 0x1ab7506cae8 }) to handler FunctionObject(FunctionObject { ptr: 0x1ab747d6188 }) : TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: text)
        at com.bamtang.swit.core::Application/loadComplete()
        at com.bamtang.swit.core::LoaderManager/loadComplete()
        at com.bamtang.swit.core::LoaderManager/onLoadImagesComplete()
        at com.bamtang.swit.core::BitmapManager/completeLoad()
        at com.bamtang.swit.core::BitmapManager/load()
        at com.bamtang.swit.core::LoaderManager/onLoadAssetComplete()
        at com.bamtang.swit.core::SwfManager/completeLoad()
        at com.bamtang.swit.core::SwfManager/load()
        at com.bamtang.swit.core::LoaderManager/onLoadDataComplete()
        at com.bamtang.swit.core::DataManager/completeLoad()
        at com.bamtang.swit.core::DataManager/load()
        at com.bamtang.swit.core::LoaderManager/fileDataLoaded()
        at com.bamtang.swit.modules::FileLoader/completeEvent()
```

### Expected behavior

The game should start. (Note: The website is using WAFlash by default, which works fine. You can switch the emulator to Ruffle using the dropdown.)

### Affected platform

Desktop app

### Operating system

Windows 10

### Additional information

This game is broken on other sites because they are missing the SWZ files; see #11344 for that issue.

**Comment by evilpie** [comment]
This now loads for me, 45 seconds after clicking play.

**Comment by n0samu** [comment]
Awesome! I believe this was fixed by #12885.
